src/devtools/client/debugger/panel.js

```diff
@@ -25,41 +24,6 @@ export class DebuggerPanel {
     return this;
   }
 
-  refresh() {
```

Comment:
Moved this to the `Devtools` component to avoid race conditions where we refresh CodeMirror before the `DevTools` component is expanded on the screen.